### PR TITLE
C#: Shorten test target names to make Windows happy.

### PR DESCRIPTION
**Pull request by criemen**
With this, we can finally re-enable the C# unit tests on Windows (no more bazel errors about path length limitations).

The target name appears 3/4 times in the relevant path, so shortening it from the original names to "t" fixes our long-path problem.

### CODE REVIEW

File: csharp/extractor/Semmle.Extraction.Tests/BUILD.bazel [hunk 1]
```diff
@@ -4,7 +4,8 @@ load(
 )
 
 codeql_xunit_test(
-    name = "Semmle.Extraction.Tests",
+    # short name as we run into long path limitations on Windows
+    name = "t",
```
Comment (michaelnebel) [inline csharp/extractor/Semmle.Extraction.Tests/BUILD.bazel:8]: Maybe use different short names for each project?

Comment (criemen) [reply csharp/extractor/Semmle.Extraction.Tests/BUILD.bazel]: Sure! Let's see if 4 characters are too much already or not.

**Comment by criemen** [comment]
It seems when I ported the test suite to Bazel, I forgot to enable `Semmle.Util.Tests`. Mea culpa!
I just discovered that, and the tests do fail on CI. I'll log a separate issue with your team about that, for now I've left the

**Review by michaelnebel — APPROVED** [review]
LGTM!
Thx!! 🎅

**Pull request merged**